custom_components/frigate/config_flow.py

```diff
@@ -115,6 +119,22 @@ def _show_config_form(
         if user_input is None:
             user_input = {}
 
+        """Handle Proxy Auth Secret authentication in the UI gracefully."""
+        use_proxy_auth_secret = user_input.get("use_proxy_auth_secret", False)
+        ignored_if_proxy_auth = (
+            "(Ignored if 'Use Proxy Auth Secret' is checked)"
+            if use_proxy_auth_secret else ""
+        )
+        ignored_if_not_proxy_auth = (
+            ""
+            if use_proxy_auth_secret else "(Ignored if 'Use Proxy Auth Secret' is checked)"
+        )
+        username_description = ignored_if_proxy_auth
+        password_description = ignored_if_proxy_auth
+        x_proxy_auth_secret_description = ignored_if_not_proxy_auth
+        x_forwarded_user_description = ignored_if_not_proxy_auth
+        x_forwarded_groups_description = ignored_if_not_proxy_auth
```

Comment:
Not a fan of this, this uses English strings in the form directly, the user may have a different language set. This should either be localized, or not used at all -- I favor the latter, this seems like it will result in UI text potentially ending up in the config entry itself.